README.md

```diff
@@ -48,7 +48,9 @@ First, you will need to customize and improve your document retrieval project on
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [nvm](https://github.com/nvm-sh/nvm#installation-and-update)
+  - Set Node version to 16.x.x (e.g. `16.14.2`) via `nvm install 16.14.2` and `nvm use 16.14.2`
```

Comment:
Nit: I think you can just do `nvm install 16` (same for `use`) and it will just use the latest.